stable_baselines/common/policies.py

```diff
@@ -123,6 +123,17 @@ def __init__(self, sess, ob_space, ac_space, n_env, n_steps, n_batch, reuse=Fals
         self.ob_space = ob_space
         self.ac_space = ac_space
 
+    @staticmethod
+    def _kwargs_check(feature_extraction, kwargs):
```

Comment:
Here it is unclear why the combination of 'mlp' feature extraction and specified kwargs is not allowed. Actually I can see myself using that combination. Please elaborate.
...
Ah while reading the rest of the diff it seemed like you are trying to get rid of the feature_extraction='mlp' option alltogether? I think this is a good idea because it was a bit redundant to begin with. But then we should mention this in the comment of this check function so future us does not get confused by this seemingly weird check.

Reply:
Good point as it is not trivial, I added a comment for that ;)

Reply:
thx